packages/integrations/image/README.md

```diff
@@ -469,7 +466,7 @@ const imageUrl = 'https://www.google.com/images/branding/googlelogo/2x/googlelog
 ```
 
 ## Troubleshooting
-- If your installation doesn't seem to be working, make sure to restart the dev server.
+- If your installation doesn't seem to be working, try restarting the dev server.
```

Comment:
Taggin @sarah11918 from my docs PR earlier. Using this verbage now!

Reply:
LGTM, @bholmesdev! 🥳